thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/business/service/WrongAnswerService.java

```diff
@@ -45,24 +44,19 @@ private List<String> getWrongAnswers(Term rightAnswerTerm) {
             throw new ExternalServiceException("Error while querying Datamuse", e);
         }
 
-        var relatedNounsSorted = relatedWords.stream()
+        var relatedNouns = relatedWords.stream()
```

Comment:
These Nouns are still sorted thought...

Reply:
Ok, I don't know why this change was not applied, they are no longer sorted